src/SimplygonFunctionApp/AzRemeshFn.cs

```diff
@@ -97,7 +106,7 @@ public static class AzRemeshFn
             };
         }
 
-        private static async Task<string> RunRemeshingAsync(Simplygon.ISimplygon sg, ILogger log, string filePath, string filePathOutput, uint onScreenSize)
+        private static async Task<string> RunRemeshingWithMaterialCastingAsync(Simplygon.ISimplygon sg, ILogger log, string filePath, string filePathOutput, uint onScreenSize)
         {
             log.LogInformation($"Scene Importer set file path: {filePath}");
             log.LogInformation($"Scene Exporter set file path: {filePathOutput}");
```

Comment:
Merge these 2 log info lines into 1 line, as suggested:

`log.LogInformation($"Scene Exporter set file path: {filePath} and output {filePathOutput}");`